examples/textual_inversion/textual_inversion_mlflow.py

```diff
@@ -143,18 +145,19 @@ def log_validation(text_encoder, tokenizer, unet, vae, args, accelerator, weight
     pipeline.set_progress_bar_config(disable=True)
 
     # run inference
-    generator = None if args.seed is None else torch.Generator(device=accelerator.device).manual_seed(args.seed)
+    generator = None if args.seed is None else torch.Generator(device=accelerator.device)
     images = []
     for _ in range(args.num_validation_images):
         with torch.autocast("cuda"):
             image = pipeline(args.validation_prompt, num_inference_steps=25, generator=generator).images[0]
+
         images.append(image)
 
     for tracker in accelerator.trackers:
```

Comment:
[Optional] Can we add `mlflow` as a tracker?

Reply:
I think we can. I'll try it